src/gpio.rs

```diff
@@ -155,7 +155,9 @@ impl GPIO {
             daisy28,
             daisy29,
             daisy30,
-        }
+        };
+        gpio.reset_codec();
+        gpio
```

Comment:
this code reset pin, isn't actually a `gpio` pin on the daisy right? maybe it should just belong to `Audio` ?

Reply:
It is.
`codec: gpio::gpiob::PB11<Output<PushPull>>,`

Reply:
But, in the context of the 'daisy seed board', this pin isn't actually available for general purpose i/o, right?
Not exposed for the user to use for any other purpose than resetting the codec?
Its not listed in: https://github.com/electro-smith/DaisyWiki/blob/master/resources/Daisy_Seed_Pinout.csv

The LED also isn't GPIO either right? Because it is hardwired to the onboard LED and not exposed for the user?

Reply:
I see what you are getting at. Yes, both of those GPIO pins are hard wired and unavailable for other purposes.

Reply:
cool.. the c++ libdaisy, doesn't include those in its list of [seedgpio](https://github.com/electro-smith/libDaisy/blob/fb2173eee60b9c5a594fbbd9911b0536c08a3610/src/daisy_seed.cpp#L17-L53)